File: dartEstagio/appDart3/class/pessoa.dart
class Pessoa {
  String _nome;
  String _sexo;
  int _idade;
  double _salario;

  Pessoa(this._nome, this._sexo, this._idade, this._salario);

  String get nome {
    return _nome;
  }

  String get sexo {
    return _sexo;
  }

  int get idade {
    return _idade;
  }

  double get salario {
    return _salario;
  }
}

File: c#Estagio/Laboratorio7/Pessoa.cs
using System;

namespace Laboratorio7
{
    public class Pessoa : IComparable<Pessoa>
    {
        private string meuNome;
        private int minhaIdade;

        public Pessoa(string n, int i)
        {
            meuNome = n;
            minhaIdade = i;
        }

        public string Nome
        {
            get { return meuNome; }
        }

        public int Idade
        {
            get { return minhaIdade; }
            set { minhaIdade = value; }
        }

        public int CompareTo(Pessoa outro)
        {
            return meuNome.CompareTo(outro.meuNome);
        }

        // public int CompareTo(Pessoa outro)
        // {
        //     return minhaIdade.CompareTo(outro.minhaIdade);
        // }
    }
}